src/wallet/rpcwallet.cpp

```diff
@@ -97,6 +98,216 @@ CBitcoinAddress GetNewAddressFromAccount(const std::string purpose, const UniVal
+                "                                                         types: nonstandard, pubkey, pubkeyhash, scripthash, multisig, nulldata, witness_v0_keyhash,\n"
+                "                                                         witness_v0_scripthash, witness_unknown.\n"
+                "  \"hex\" : \"hex\",                      (string, optional) The redeemscript for the p2sh address.\n"
+                "  \"pubkeys\"                           (array, optional) Array of pubkeys associated with the known redeemscript (only if script is multisig).\n"
+                "    [\n"
+                "      \"pubkey\" (string)\n"
+                "      ,...\n"
+                "    ]\n"
+                "  \"sigsrequired\" : xxxxx              (numeric, optional) The number of signatures required to spend multisig output (only if script is multisig).\n"
+                "  \"pubkey\" : \"publickeyhex\",          (string, optional) The hex value of the raw public key for single-key addresses (possibly embedded in P2SH or P2WSH).\n"
+                "  \"embedded\" : {...},                 (object, optional) Information about the address embedded in P2SH or P2WSH, if relevant and known. Includes all\n"
+                "                                                         getaddressinfo output fields for the embedded address, excluding metadata (timestamp, hdkeypath,\n"
+                "                                                         hdseedid) and relation to the wallet (ismine, iswatchonly).\n"
+                "  \"iscompressed\" : true|false,        (boolean, optional) If the pubkey is compressed.\n"
+                "  \"label\" :  \"label\"                  (string) The label associated with the address. Defaults to \"\". Equivalent to the label name in the labels array below.\n"
+                "  \"timestamp\" : timestamp,            (number, optional) The creation time of the key, if available, expressed in the UNIX epoch time.\n"
+                "  \"hdkeypath\" : \"keypath\"             (string, optional) The HD keypath, if the key is HD and available.\n"
+                "  \"hdseedid\" : \"<hash160>\"            (string, optional) The Hash160 of the HD seed.\n"
+                "  \"hdmasterfingerprint\" : \"<hash160>\" (string, optional) The fingerprint of the master key.\n"
+                "  \"labels\"                            (json object) An array of labels associated with the address. Currently limited to one label but returned\n"
+                "                                               as an array to keep the API stable if multiple labels are enabled in the future.\n"
+                "    [\n"
+                "      \"label name\" (string) The label name. Defaults to \"\". Equivalent to the label field above.\n\n"
+                "      DEPRECATED, will be removed in 0.21. To re-enable, launch bitcoind with `-deprecatedrpc=labelspurpose`:\n"
```

Comment:
launch **pivxd** with.
And actually, do we really want to inlcude deprecated fields? In that case the version should be adjusted. Maybe 5.0.